test/cli/run.sh

```diff
@@ -13,7 +13,7 @@ docker volume create app_data
 # correctly. It is not supposed to hold any data.
 docker volume create empty_data
 
-docker run -d \
+docker run -d -q \
```

Comment:
At least on my setup, `docker run --help` doesn't tell me that there's a `-q` option. What is it supposed to do?

Reply:
It's short for --quiet (see [docs](https://docs.docker.com/engine/reference/commandline/run/)) and like --quiet-pull it doesn't print out the progress bar of the image layer download & extraction which spams the CI logs with 10-50 lines each time. It still prints what its doing though, especially container startup etc., only affects the progress bar of the pull.